client/checkout/woopay/email-input-iframe.js

```diff
@@ -546,13 +529,13 @@ export const handlePlatformCheckoutEmailInput = async (
 				hasCheckedLoginSession = true;
 				closeLoginSessionIframe();
 				break;
-			case 'redirect_to_platform_checkout':
+			case 'redirect_to_woopay':
 				wcpayTracks.recordUserEvent(
-					wcpayTracks.events.PLATFORM_CHECKOUT_OTP_COMPLETE
+					wcpayTracks.events.WOOPAY_OTP_COMPLETE
 				);
-				api.initPlatformCheckout(
-					platformCheckoutEmailInput.value,
-					e.data.platformCheckoutUserSession
+				api.initWooPay(
+					woopayEmailInput.value,
+					e.data.woopayUserSession
```

Comment:
Same issue with keeping `e.data.platformCheckoutUserSession`.

Reply:
Fixed in https://github.com/Automattic/woocommerce-payments/pull/6061/commits/5dade92dafb2f7f088cff7f5d3408af76e7b40f6